src/PointerType.ts

```diff
@@ -3,4 +3,5 @@ export enum PointerType {
   STYLUS,
   MOUSE,
   OTHER,
+  KEY,
```

Comment:
I'd swap the order and leave `OTHER` as last element.

Reply:
This order is intentional. I've seen this enum used on the native side, and I'm not sure web and native side interface `PointerType` at some point. 

I'm also not sure if this enum isn't based on some sort of built-in enum, which may directly interface with this one.
 
I'll have to double-check none of these cases occur before applying this change.
For now, keeping `KEY` on the last position ensures no such interference to occur.

Reply:
> I'm also not sure if this enum isn't based on some sort of built-in enum, which may directly interface with this one.

It's not 😅 But good catch that the order might matter. I'm not sure if that would be better to keep it, or change the order and then make changes in relevant places (cc @j-piasecki)

Reply:
Couldn't find any causes for concern, and so i applied your request in [aff6119](https://github.com/software-mansion/react-native-gesture-handler/pull/3035/commits/aff611955e6e7b0ae663f4f461becc9fded0dd28)